etc/ci/performance/test_perf.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
Please set the `set -o *` lines as described in the [style guide](https://github.com/servo/servo/blob/master/docs/STYLE_GUIDE.md#shell-scripts)